spec/delivery_task_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'rspec'
+require './delivery_task.rb'
+require 'vcr'
+require 'pry'
+
+VCR.configure do |c|
+	c.cassette_library_dir = "spec/vcr"
+	c.hook_into :webmock
+	vcr_mode = :once
+end
+
+RSpec.describe Delivery do
+
+	describe '#routs' do
```

Comment:
Class methods should be described as ".routs", only instance methods are described as "#routs"